indico/MaKaC/webinterface/pages/category.py

```diff
@@ -431,56 +395,33 @@ def _getChairText( self ):
             chairs = "(%s)"%"; ".join(l)
         return chairs
 
-    def _getLocation( self ):
-        loc = ""
-        if self._conf.getLocation() != None:
-            loc = self._conf.getLocation().getName()
-        room = ""
-        if self._conf.getRoom() != None and self._conf.getRoom().getName() != None:
-            room = self._conf.getRoom().getName()
-            url = "javascript:redirectToRoomLoc('" + escape_html(room) + "','" + escape_html(loc) +"')"
-            loc = "%s: "%loc
-            room = """<a href="%s" style="font-size: 0.9em">%s</a>"""%(url, room)
-        if loc != "" or room != "":
-            return "(%s%s)"%(loc, room)
-        else:
-            return ""
-
     def _getBreakItem( self, entry ):
         wc = WOverviewBreak( self._aw, entry )
         return wc.getHTML( {} )
 
-    def _getDetails( self ):
+    def _getDetails(self):
         if self._details == "conference":
             return ""
         res = []
-        tz = DisplayTZ(self._aw,useServerTZ=1).getDisplayTZ()
-        for entry in self._conf.getSchedule().getEntriesOnDay( self._date ):
-            if isinstance( entry, schedule.LinkedTimeSchEntry) and \
-                        isinstance( entry.getOwner(), conference.Session ) and entry.getOwner().canView(self._aw):
-                wc = WOverviewSession( self._aw, \
-                                        entry.getOwner(), \
-                                        self._date, \
-                                        self._details )
-                res.append( wc.getHTML({}) )
-            elif isinstance( entry, schedule.LinkedTimeSchEntry) and \
-                        isinstance( entry.getOwner(), conference.SessionSlot ) and entry.getOwner().canView(self._aw):
-                wc = WOverviewSessionSlot( self._aw, \
-                                        entry.getOwner(), \
-                                        self._date, \
-                                        self._details )
-                res.append( wc.getHTML({}) )
+
+        for entry in self._conf.as_event.timetable_entries.options(joinedload('children')).filter(
+                TimetableEntry.parent_id == None,
```

Comment:
`TimetableEntry.parent_id.is_(None)` does the same but won't trigger pep8 notices about `== None`